### DISCUSSION TITLE
guide-R/donnees_ponderees/analyses-bivariees

### DISCUSSION THREAD

**Discussion by giscus** [larmarange/guide-R#48] [Forum]
# guide-R/donnees_ponderees/analyses-bivariees

https://larmarange.github.io/guide-R/donnees_ponderees/analyses-bivariees.html

**Comment by N0unouille** [comment]
Bonjour Mr, 
Je cherche à faire un tri à plat de plusieurs variables, prenons par exemple data$SEXE, avec pondération grâce à une variable IPONDI, mais je n'y parviens pas. Auriez vous une piste ou une fonction à me conseiller ? 
Merci d'avance

**Reply by larmarange** [reply]
Je vous invite à lire en premier lieu ce chapitre https://larmarange.github.io/guide-R/donnees_ponderees/plan-echantillonnage.html
qui vous permettra de définir votre pondération, avant de pouvoir utiliser tbl_svysummary()
